sync/src/block/mod.rs

```diff
@@ -19,3 +19,9 @@ mod extension;
 mod message;
 
 pub use self::extension::Extension as BlockSyncExtension;
+
+use cnetwork::NodeId;
+
+pub trait BlockSyncForRPC: Send + Sync {
```

Comment:
How about renaming it to `BlockSyncInfo`

Reply:
Good!